pkg/testcase/localpathstorage.go

```diff
@@ -67,9 +63,11 @@ func readData() error {
 	if err != nil {
 		return err
 	}
-	time.Sleep(160 * time.Second)
 
-	fmt.Println("Read data from newly create pod")
+	fmt.Println("Reading data from newly created pod")
+	delay := time.After(30 * time.Second)
```

Comment:
Do we have to wait 30 seconds here?

Reply:
Is there a way to add a common retry/polling logic to handle situations like this?

Reply:
- Do we have to wait 30 seconds here?
```
 we need yeah, is the minimun i found to the pod get "recreated"
```

- Is there a way to add a common retry/polling logic to handle situations like this dynamically? Same suggestion for any other place delay is used
```
 for sure, i just dont know a better solution yet that is not  too complex and its also concise.
```

Reply:
Probably worth adding a task to handle this type of situation for dynamically